scripts/download_webcams.sh

```diff
@@ -8,5 +8,7 @@ set -x
 find /var/webcam -mtime +$IPL_WEBCAM_KEEP_DAYS -type f -delete
 # Delete all empty directories
 find /var/webcam  -type d -empty -delete
-# Download all new files
-lftp -e "mirror -c --parallel=$IPL_WEBCAM_WORKER --verbose / /var/webcam; quit;" -u $IPL_WEBCAM_USER,$IPL_WEBCAM_PASSWORD $IPL_WEBCAM_SERVER
+# Download all new files. Limit to now -(DOWNLOAD_MAX_DAYS - 1) days, otherwise we would delete and re-download data
```

Comment:
Can you elaborate a bit more? Is this because because `now - 6 days` effectively describes a time span of 7 days because the calculation is only precise to the day?